Merch.php

```diff
@@ -0,0 +1,259 @@
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-3">
+                    <div class="card">
+                        <img src="images/merch_apparel_07.jpg" class="card-img-top" alt="Longsleeve">
+                        <div class="card-body">
+                            <h4 class="card-title">Black Long Sleeve Shirt</h4>
+                            <p class="card-text">V-Neck, L $25</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-3">
+                    <div class="card">
+                        <img src="images/merch_apparel_08.jpg" class="card-img-top" alt="Shorts">
+                        <div class="card-body">
+                            <h4 class="card-title">BasketBall Shorts</h4>
+                            <p class="card-text">Available in Blue or Black, L-XL $15</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-6 my-3">
+                <h2 class="merch-title">Order online, pick up curbside</h2>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-12 col-lg-6 mb-3">
+                <div class="card">
+                    <img src="images/merch_order_ea.jpg" class="card-img-top card-order" alt="ea merch">
+                    <div class="card-body">
+                        <h4 class="card-title">East Aurora Merch</h4>
+                        <a href="https://www.toasttab.com/bar-bill/v3" class="btn btn-primary">East Aurora Takeout</a>
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm-12 col-lg-6 mb-3">
+                <div class="card">
+                    <img src="images/merch_order_cl.jpg" class="card-img-top card-order" alt="cl merch">
+                    <div class="card-body">
+                        <h4 class="card-title">Clarence Merch</h4>
+                        <a href="https://www.toasttab.com/bar-bill-north/v3" class="btn btn-primary">Clarence Takeout</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <footer class="container-fluid">
+        <div class="footer">
+            <h2>Our Locations</h2>
+            <p><em>East Aurora Takeout</em><br />
+                191 Main Street, Rear<br />
+                East Aurora, NY 14052<br />
+                (716) 652-7959
+            </p>
+            <p><em>East Aurora</em><br />
+                185 Main Street<br />
+                East Aurora, NY 14052<br />
+                (716) 652-7959
+            </p>
+            <p><em>Clarence</em><br />
+                8326 Main Street<br />
+                Clarence, NY 14221<br />
+                (716) 710-9464
```

Comment:
I would recommend looking into making these a little more interactive, maybe a google maps link or adding a `tel` tag... Just a suggestion

Reply:
That was what I was thinking of once I incorporated a working navigation header. Incorporate a common footer file that would link the addresses to Google maps.